test/modules/core/lib/attribute/attribute-manager.spec.js

```diff
@@ -295,36 +291,3 @@ test('AttributeManager.invalidate', t => {
 
   t.end();
 });
-
-test('AttributeManager.setDefaultLogFunctions', t => {
```

Comment:
Assume the new system should be nicely testable by injecting test assertions instead of log functions? Assume that can be improved especially if this becomes the foundation of a plugin system.